src/api/flaskr/service/learn/runscript_v2.py

```
@@ -45,6 +46,87 @@
+
+        if redis_client is None:
+            return True  # fail open when Redis is unavailable
+        result = redis_client.eval(
+            _LUA_ACQUIRE_ASK_SLOT,
+            1,
+            _get_ask_sem_key(app, user_bid, outline_bid),
+            str(MAX_PARALLEL_ASK_COUNT),
+            str(RUN_SCRIPT_TIMEOUT_SECONDS),
+        )
+        return bool(result)
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)